include/m_common_data.h

```diff
@@ -161,6 +161,19 @@ typedef struct FamicomEmuCommonData {
 
 typedef void (*CommonData_100E4_Func)(struct Game_Play*);
 
+typedef struct {
+    /* 0x0000 */ UNK_TYPE1 unk_0[0x14];
+    /* 0x0014 */ UNK_TYPE unk_14;
+    /* 0x0018 */ UNK_TYPE1 unk_18[0xF404];
+    /* 0xF41C */ u8 snowmanData[0xC];
+    /* 0xF428 */ u8 unk_F428[0x485];
+    /* 0xF8AD */ u8 snowmanYear;
+    /* 0xF8AE */ u8 snowmanMonth;
+    /* 0xF8AF */ u8 snowmanDay;
+    /* 0xF8B0 */ u8 snowmanHour;
+    /* 0xF8B1 */ u8 unk_F8B1[0x74F];
+} Save; // size = 0x10000
```

Comment:
What is this struct?

Reply:
It's the save structure, it's a substructure in common_data and I believe it's used and referenced in some files
I can investigate and give you a better reply later lol

Reply:
oh heck i think i made this a long time ago and didn't notice it was still in this pr, i'll remove it